scalameta/dialects/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -122,6 +127,7 @@ package object dialects {
     def allowOrTypes = Scala210.allowOrTypes
     def allowTraitParameters = Scala210.allowTraitParameters
     def allowLiteralTypes: Boolean = Scala210.allowLiteralTypes
+    def allowTrailingComma: Boolean = Scala210.allowLiteralTypes
```

Comment:
There will be a custom Sbt100 (or something like that, maybe `Sbt1`) dialect where we can set all the fields as desired.